osmosis-1/osmosis.zone_assets.json

```diff
@@ -5039,6 +5039,17 @@
         "defi",
         "built_on_osmosis"
       ]
+    },
+    {
+      "chain_name": "oraichain",
+      "base_denom": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/ton",
+      "path": "transfer/channel-216/factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/ton",
+      "osmosis_verified": false,
+      "osmosis_unstable": true,
+      "listing_date_time_utc": "2024-08-30T00:33:00Z",
+      "categories": [
+        "defi"
+      ]
```

Comment:
When the URL is confrimed, please add the transfer method (or at least let me know what the URL will be) so users can bridge it between The Open Network and Osmosis. Something like:
```
"transfer_methods": [
  {
    "name": "OraiDEX Ton Bridge",
    "type": "external_interface",
    "deposit_url": "https://ton.oraidex.io/?from=TON&to=OSMOSIS",
    "withdraw_url": "https://ton.oraidex.io/?from=OSMOSIS&to=TON"
  }
]
```